.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "mrc-dev",
+    "image": "mrc:dev",
+    "runArgs": [
+        "--gpus=all",
```

Comment:
Should also add `--runtime=nvidia`.

Reply:
The examples seem to use one or the other.

https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/user-guide.html

Reply:
I've tried with `--gpus=all` and it didn't give me `nvidia-smi`

Reply:
I think some installations of the nvidia container toolkit, particularly those opinionated machines like dgx will set this as the default runtime. 

I've added it to be explicit for those setups that do not have it set as the default.